connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Herder.java

```diff
@@ -82,7 +82,7 @@
     void connectorConfig(String connName, Callback<Map<String, String>> callback);
 
     /**
-     * Set the configuration for a connector. This supports creation, update, and deletion.
+     * Set the configuration for a connector. This supports creation and updating.
```

Comment:
Makes sense. So do we want to continue to allow `ConnectorsResource.putConnectorConfig()` to support deletion? I think I'd probably prefer to require the user call DELETE explicitly, but I'm not sure if anyone depends on this behavior at the moment.